stubs/pika/pika/compat.pyi

```diff
@@ -0,0 +1,44 @@
+from abc import ABCMeta
+from io import StringIO as StringIO
+from re import Pattern
+from typing_extensions import Final, Literal
+from urllib.parse import parse_qs, quote, unquote, urlencode as urlencode, urlparse as urlparse
+
+url_quote = quote
+url_unquote = unquote
+url_parse_qs = parse_qs
+
+PY2: Final[Literal[False]]
+PY3: Final[Literal[True]]
+RE_NUM: Final[Pattern[str]]
+ON_LINUX: Final[bool]
+ON_OSX: Final[bool]
+ON_WINDOWS: Final[bool]
```

Comment:
I guess you could plausibly want to monkey-patch these for testing purposes, which would make the `Final[]` annotation a little annoying?

Reply:
On the other hand, outside of testing, overriding these would potentially be fatal. Even for testing, changing these is potentially fragile.

Reply:
I don't have a strong opinion — it was just a thought :) feel free to ignore